src/Aspire.Hosting.Azure.Sql/AzureSqlDatabaseResource.cs

```diff
@@ -32,6 +42,16 @@ public class AzureSqlDatabaseResource(string name, string databaseName, AzureSql
     /// </summary>
     public string DatabaseName { get; } = ThrowIfNullOrEmpty(databaseName);
 
+    /// <summary>
+    /// Gets or Sets the database SKU name
+    /// </summary>
+    internal string SkuName
```

Comment:
This should be public

Reply:
I made the get public and kept the set internal, as we want to make sure developers use the `WithSku` method to set the sku, right?